docs/CONTRIBUTING.md

```diff
@@ -10,14 +10,14 @@
 - [ ] Run `npm run build:reset` to remove changes to compiled files.
 - [ ] Submit a Pull Request.
 
-## Design principles
+<h2 id="design-principles">Design principles</h2>
```

Comment:
Should we enable header ids instead of doing this manually?

Reply:
We are already doing it manually in most of our pages so the ids don't get too long. I think we should keep it consistent and we don't want to change it for existing ids otherwise urls will break.